awx/main/models/events.py

```diff
@@ -151,6 +151,10 @@ class Meta:
         blank=True,
         default={},
     )
+    profiling_data = JSONField(
```

Comment:
If anyone has a preference on what to name this field give a shout (cc @wenottingham)

Reply:
🤷‍♂️ works for me

Reply:
`    📉📈 = JSONField(` ...

Reply:
![](https://thisfangirlsperspective.files.wordpress.com/2014/06/spider-man-2-uncle-ben-cliff-robertson-great-power-quote.png)

Reply:
Preach, Uncle Ben

Reply:
@chrismeyersfsu a part of me wonders if we should make this a JSONB field, but I also think that querying this in-process in AWX is probably a bad idea.  This is the type of data we'd likely want to export and analyze _en masse_ in some _other_ tooling.